gradle/dependencies.gradle

```diff
@@ -11,7 +11,7 @@ ext {
   // version which we should use in this build
   def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
   if (mapboxNavigatorVersion == null || mapboxNavigatorVersion == '') {
-      mapboxNavigatorVersion = '48.0.5'
+      mapboxNavigatorVersion = 'da-common-runloop-callback-11-SNAPSHOT'
```

Comment:
@Guardiola31337 is this a real release yet?

Reply:
Nope, not yet. This will be included in the next NN stable release. We also want to integrate and test from our side first.

cc @DmitryAk @mskurydin